### ISSUE TITLE
Choose Head Size Based off of age of Participant

### ISSUE THREAD

**Issue by audreymhoughton** [nipreps/nibabies#198]
### A short summary of what you would like to see in *NiBabies*.

Scale brain size flag based off of the age of the participant. Will need to find or make a table for the best brain size for each age.

### Do you have any interest in helping implement the feature?

Yes, but I would need guidance

### Add any additional information or context about the request here.

I think we should be able to help find / make a table. We should have a resource we can use through the BCP data. I don't have the bandwidth to help add into the code itself though.

**Comment by tsalo** [comment]
What about calculating the head size from the anatomical image, as we do in XCP-D?

**Comment by mgxd** [comment]
@tsalo are you referring to [this function](https://github.com/PennLINC/xcp_d/blob/1d7bce24ae70f955d64641afbb75226267c3fd27/xcp_d/utils/utils.py#L286)? Looks interesting and reasonable to add

**Comment by tsalo** [comment]
Absolutely! There are probably better ways to do it than assuming the brain is a sphere, but that function seems to work fairly well (not that I've done rigorous testing).
